src/components/PeopleFilters/components/CenturyFilter.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import React, { useCallback } from 'react';
+import { getSearchWith } from '../../../utils/searchHelper';
+import { updateSearchParams } from '../../PeopleTable/utils/updateSearchParams';
+
+const CenturyFilters = [16, 17, 18, 19, 20];
```

Comment:
```suggestion
const CenturyFilters = [16, 17, 18, 19, 20];
```
in camelCase or in UPPER_CASE if it's constant